apiserver/root.go

```diff
@@ -33,6 +33,7 @@ import (
 	"github.com/juju/juju/internal/migration"
 	"github.com/juju/juju/internal/rpcreflect"
 	"github.com/juju/juju/internal/servicefactory"
+	workerobjectstore "github.com/juju/juju/internal/worker/objectstore"
```

Comment:
The interface you're pulling from here should go to core, so you don't need this import.